tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/builtinconstraints/EmailConstraintTest.java

```diff
@@ -49,17 +47,11 @@ public static WebArchive createTestArchive() {
 	@SpecAssertion(section = Sections.BUILTINCONSTRAINTS, id = "v")
 	public void testEmailConstraint() {
 		Validator validator = TestUtil.getValidatorUnderTest();
-		EmailDummyEntity foo = new EmailDummyEntity();
+		EmailDummyEntity foo = new EmailDummyEntity( "invalid" );
 
 		Set<ConstraintViolation<EmailDummyEntity>> constraintViolations = validator.validate( foo );
-		assertNumberOfViolations( constraintViolations, 0 );
-
-		foo.email = "4 8";
-		constraintViolations = validator.validate( foo );
-		assertCorrectConstraintTypes( constraintViolations, Email.class );
-		assertThat( constraintViolations ).containsOnlyPaths(
-				pathWith()
-						.property( "email" )
+		assertThat( constraintViolations ).containsOnlyViolations(
```

Comment:
Outch, yes, that's not what I wanted to do... It should have been a proper email.